engine/src/main/java/org/camunda/bpm/engine/repository/ProcessDefinitionQuery.java

```diff
@@ -78,10 +78,21 @@ public interface ProcessDefinitionQuery extends Query<ProcessDefinitionQuery, Pr
   ProcessDefinitionQuery processDefinitionKey(String processDefinitionKey);
 
   /**
-   * Only select process definitions with the given keys
+   * Only select process definitions with the given keys.
+   * <p><b>Deprecated</b>: use {@link #processDefinitionKeyIn(String...)}</p>
    */
+  @Deprecated(since = "7.20.1")
```

Comment:
As the change will not be backported, the correct version for deprecation will be `7.21.0`. Please adjust the `since` value or remove it.
```suggestion
  @Deprecated
```